src/HBL01MItineraryHashtag/HBL01MItineraryHashtag.controller.ts

```diff
@@ -0,0 +1,14 @@
+import { Controller, Get } from '@nestjs/common';
+
+import { HBL01MItineraryHashtagService } from './HBL01MItineraryHashtag.service';
+
+@Controller('HBL01')
+export class HBL01MItineraryHashtagController {
+  constructor(private readonly service: HBL01MItineraryHashtagService) {}
+
+  // curl -XGET -H "Content-Type:application/json" localhost:3000/HBL01
+  @Get()
+  HBL01() {
+    return this.service.doExecute();
+  }
```

Comment:
更新処理だからPOSTの方がいいと思う！（将来的にAPIKeyとか付けれるようになるから！）
```suggestion
  @Post()
  HBL01(@Body() HBL01Body: Record<string, never>) {
    return this.service.doExecute();
  }
}
```

Reply:
[dcde40c](https://github.com/Ayato-kosaka/spelieve-backend-tag/pull/3/commits/dcde40c3e7f40def062a68352a2177bfea9096db)